modules/apps/layout/layout-taglib/src/main/java/com/liferay/layout/taglib/servlet/taglib/renderer/LayoutStructureRenderer.java

```diff
@@ -210,7 +212,22 @@ private void _renderCollectionStyledLayoutStructureItem(
 
 		JspWriter jspWriter = _pageContext.getOut();
 
-		jspWriter.write("<div class=\"");
+		jspWriter.write("<div");
+
+		if (FeatureFlagManagerUtil.isEnabled("LRAC-14922")) {
+			ListObjectReference listObjectReference =
+				renderCollectionLayoutStructureItemDisplayContext.
+					getListObjectReference();
+
+			if (listObjectReference != null) {
+				jspWriter.write(" analytics-targetable-collection=\"");
```

Comment:
Hi @ealonso, can you rename it?
```suggestion
				jspWriter.write(" data-analytics-targetable-collection=\"");
```

This way we will be able to get data from js like: `foo.dataset.analyticsTargetableCollection`